pinot-controller/src/main/java/com/linkedin/pinot/controller/validation/ValidationManager.java

```diff
@@ -180,6 +192,22 @@ public void runValidation() {
     LOGGER.info("Validation completed");
   }
 
+  // If we add a new broker, we want to rebuild the broker resource.
+  void rebuildBrokerResourceWhenBrokerAdded(AbstractTableConfig tableConfig, IdealState brokerIdealState) {
```

Comment:
why do we need a new method for this?. We should simply trigger the same code to generate brokerIdealState based on the tenant configs right?